### PR TITLE
Fix release builds

### PR DESCRIPTION
**Pull request by tasomaniac**
This will fix proguard and multidex issues we are having recently. 

#572 removes GCM and this removes location play services dependency. Once they are removed, we will be at 57K out of 65K method limit. 

After these 2 are merged, we can publish an alpha release.

### CODE REVIEW

File: app/google-services.json [hunk 1]
```diff
@@ -1 +1,165 @@
...
\ No newline at end of file
+{
```
Comment (tasomaniac) [inline app/google-services.json]: Seems huge but this is generated by Google.

File: app/src/main/res/values/analytics.xml [hunk 2]
```diff
@@ -1,25 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?><!--
-  ~ Copyright 2013-2015 The GDG Frisbee Project
```
Comment (tasomaniac) [inline app/src/main/res/values/analytics.xml]: These are coming from Google Api Gradle plugins automatically

File: build.gradle [hunk 3]
```diff
@@ -7,7 +7,7 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:2.0.0-beta6'
-        classpath 'com.google.gms:google-services:1.3.1'
+        classpath 'com.google.gms:google-services:2.0.0-beta6'
```
Comment (friedger) [inline build.gradle]: should be 2.1.0-alpha1 :-)

File: app/build.gradle [hunk 4]
```diff
@@ -64,12 +64,10 @@ android {
             proguardFile '../settings/proguard/proguard-square-retrofit2.pro'
             proguardFile '../settings/proguard/proguard-butterknife.pro'
             proguardFile '../settings/proguard/proguard-crashlytics.pro'
-            proguardFile '../settings/proguard/proguard-joda-time.pro'
```
Comment (friedger) [inline app/build.gradle]: Why is this gone? We shouldn't use joda anyway.

Comment (tasomaniac) [inline app/build.gradle]: JodaTime now includes its proguard configuration automatically through Gradle

File: app/build.gradle [hunk 5]
```diff
@@ -142,9 +140,7 @@ dependencies {
     debugCompile 'com.android.support:multidex:1.0.1'
```
Comment (friedger) [inline app/build.gradle]: We still need multidex support then?

File: build.gradle (hunk 3 above)
Comment (tasomaniac) [inline build.gradle]: I didn't do it because some people may not prefer the preview release

File: app/build.gradle (hunk 5 above)
Comment (tasomaniac) [inline app/build.gradle]: Multidex is only on debug because we don't use proguard there.

**Comment by tasomaniac** [comment]
What is holding you @friedger  :smile: I really want to release an alpha build by pushing to master.

**Comment by friedger** [comment]
:+1:

**Comment by tasomaniac** [comment]
LGTM

**Comment by tasomaniac** [comment]
Now it can be merged, apparently this should be done after the last commit.

**Pull request merged**